lib/contexts/DiceContext/DiceContext.tsx

```diff
@@ -26,18 +24,18 @@ export const DefaultDiceCommandOptions: Array<IDiceCommandOption> = [
 ];
 
 export function useDice() {
-  const [latestCommandOptions, setLatestCommandOptions] = useState<
-    Array<IDiceCommandOption>
-  >(DefaultDiceCommandOptions);
```

Comment:
Don't want to use latest rolled commands anymore, but instead just want to always roll what ever command groups are selected in the menu